README.md

```diff
@@ -16,43 +16,70 @@ Independent analyzers are executed with their module name, e.g. `python3.7 -m un
-2. Python 3.7
-3. Pip for python3.7
+2. Python version 3.7 or greater
+3. Pip version 9.0 or greater
 ```bash
 sudo pip3.7 install -U universum
 ```
 or
 ```bash
 pip3.7 install --user -U universum
 ```
+Can also be installed with [extras for using VCS](
+https://universum.readthedocs.io/en/latest/install.html#vcs-related-extras),  but they also require
+installing respective command-line tools, such as git or p4.
 
 ### Latest development + tests
 
-Additional prerequisites ([see documentation for details](
-https://universum.readthedocs.io/en/latest/prerequisites.html#optional-used-for-internal-tests)):
-1. Perforce CLI, P4Python (['helix-cli' and 'perforce-p4python'](
-https://www.perforce.com/manuals/p4sag/Content/P4SAG/install.linux.packages.install.html))
-2. Docker (['docker-ce', 'docker-ce-cli'](
-https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce))
-3. Current user added to 'docker' group (use `sudo usermod -a -G docker $USER` and then relogin)
+Additional prerequisites for testing Universum locally:
+1. Support of all [VCS extras](https://universum.readthedocs.io/en/latest/install.html#vcs-related-extras),
+   including installation of Git and P4 CLI
+2. Manual installation of Docker (`docker-ce`, `docker-ce-cli`) (see [official installation manual](
+   https://docs.docker.com/engine/installation/linux/ubuntu/#install-using-the-repository) for details)
+
+   * Also add current user to 'docker' group (use `sudo usermod -a -G docker $USER` and then relogin)
 ```bash
+python3.7 -m venv virtual-environment-python3.7
+source ./virtual-environment/bin/activate
 git clone https://github.com/Samsung/Universum.git universum-working-dir
 cd universum-working-dir
 git checkout master
-pip3.7 install --user -U .[test]
+pip install -U .[test]
 make images
+make test
 ```
-After this run `make tests` and ensure all tests are passing.
 
-Also note that running `pip uninstall universum` will remove Universum itself,
-but all the dependency modules will remain in the system.
+The `[test]` extension will install/update the following additional Python modules:
+
+    * `sphinx`
+    * `sphinx-argparse` (extension for `Sphinx`)
+    * `sphinx_rtd_theme` (extension for `Sphinx`)
+    * `pytest`
+    * `pylint`
+    * `docker`
+    * `httpretty`
+    * `mock`
+
+Although it is possible to get these modules via `pip3.7 install -U universum[test]`, it might be more convenient
+to checkout the Universum branch you are currently working on, change working directory to project root and
+run a `pip3.7 install -U .[test]` command from there for more flexibility. Using virtual environment (via `venv`
+command) allows to separate test environment from system and provides even more control over additional modules.
+
+Uninstalling Universum via `pip uninstall universum` will not uninstall all the dependencies installed along with it.
+Simply deleting the created virtual environment will leave the system completely cleaned of all changes.
```

Comment:
I think "Simply deleting the directory with virtual environment" would be more explicit and developer-friendly.